src/dataflow-types/src/types.rs

```diff
@@ -1983,6 +1985,88 @@ pub mod sources {
         },
     }
 
+    impl Arbitrary for DebeziumMode {
```

Comment:
why cant this be derived? the datetime stuff?

Reply:
Exactly, `NaiveDateTime` comes directly from `chrono` and doesn't implement `Arbitrary`, so we have to use our manual generators for it.